mlrun/feature_store/feature_vector.py

```diff
@@ -493,11 +501,6 @@ def get(self, entity_rows: List[Union[dict, list]], as_list=False):
         for row in entity_rows:
             futures.append(self._controller.emit(row, return_awaitable_result=True))
 
-        requested_columns = list(self.vector.status.features.keys())
-        aliases = self.vector.get_feature_aliases()
-        for i, column in enumerate(requested_columns):
-            requested_columns[i] = aliases.get(column, column)
-
```

Comment:
Looks like you changed the behavior here - if `requested_columns` is `None` this used to contain the list of all features in the fvec. However, now you keep this list empty, which seems to me that will cause a different behavior - when initializing without specifying requested columns you'll get back nothing. 
Am I missing something here?

Reply:
@theSaarco The aliasing is preformed in the graph (in `QueryByKey` steps) and what you mentioned executed few lines down